Makefile-rpm-ostree.am

```diff
@@ -57,3 +57,10 @@ rpm_ostree_LDADD = $(PKGDEP_RPMOSTREE_LIBS) librpmostreepriv.la librpmostree-1.l
 
 privdatadir=$(pkglibdir)
 privdata_DATA = src/app/rpm-ostree-0-integration.conf
+
+install-bin-hook:
+	mv $(DESTDIR)$(bindir)/rpm-ostree $(DESTDIR)$(bindir)/$(primaryname)
+	ln -sf $(primaryname) $(DESTDIR)$(bindir)/rpm-ostree
+if BUILDOPT_NEW_NAME
+INSTALL_DATA_HOOKS += install-bin-hook
```

Comment:
Though this should be ultimately be using `install-exec-hook`, no? So i.e. have a new analogous `INSTALL_EXEC_HOOKS`? But meh, not sure in what circumstances this would actually matter.